### DISCUSSION TITLE
Version 1.18.0

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#191] [Announcements]
- We now have a website with a [user guide](https://kvaesitso.mm20.de/docs/user-guide/)!
  - relevant user guide pages are linked in launcher settings
- New default typeface (Poppins has been replaced with Outfit)
- Added a dark variant to search bar and clock widget. Search bar, clock widget and system bars will now automatically turn dark when you use a bright wallpaper. You can also enable the dark variant for each of these components individually
- Added Pebble icon shape
- Added 1 new clock style
- Improved app search matching; diacritics will now be ignored
- Keyboard will now close when scrolling the search result list
- Added missing surface tint color to advanced custom color scheme
- Bugfixes and minor improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.18.0'>Version 1.18.0</a>.</em>

**Comment by Tealk** [comment]
The new watch is really cool

**Comment by Drsheppard01** [comment]
Is it possible to leave Poppins as an additional typeface?

**Reply by MM2-0** [reply]
I remove it because each typeface adds to the APK size and both typefaces are pretty similar (they are both geometric sans serif fonts) but if there is demand, I can bring it back.
